### ISSUE TITLE
bbox expects a value

### ISSUE THREAD

**Issue by pnorman** [kosmtik/kosmtik#118]
```
$ node index.js export ~/osm/osm-clear/project.yaml --width 768 --height 768 --output img.png --bbox -14401959.12137977,4383204.949985144,-11897270.57853111,6887893.492833805

[Core] No usable config file found in /home/pnorman/.config/kosmtik.yml
[Core] Loading plugin from ../plugins/base-exporters/index.js
[Core] Loading plugin from ../plugins/hash/index.js
[Core] Loading plugin from ../plugins/local-config/index.js
[Core] Loading plugin from ../plugins/datasource-loader/index.js
'--bbox' expects a value


Usage: node index.js export [project] [options]
...
```

I also tried

* `--bbox '-14401959.12137977,4383204.949985144,-11897270.57853111,6887893.492833805'`
* `'--bbox -14401959.12137977,4383204.949985144,-11897270.57853111,6887893.492833805'`
* `--bbox '-14401959.12137977','4383204.949985144','-11897270.57853111','6887893.492833805'`

**Comment by yohanboniface** [comment]
Humm, I wonder if it's related to the value starting with `-`, cf #117 
Can you try with `--bbox=-144…`?

(Also, that does not sound like WGS84 ;) )

**Comment by pnorman** [comment]
> Also, that does not sound like WGS84

Nope, bbox talks of X and Y, and this project is in 900913, so there's no reason to expect WGS84 in the documentation

``--bbox=-14401959.12137977,4383204.949985144,-11897270.57853111,6887893.492833805`` works

**Comment by yohanboniface** [comment]
> --bbox=-14401959.12137977,4383204.949985144,-11897270.57853111,6887893.492833805 works

nomnom is certainly taking the first `-` as a parameter argument sign. Not easy to fix I think, so maybe the `=` is mandatory in any case. I'll dig on it.

**Comment by pnorman** [comment]
I looked at the rendering result from it, and it's the whole world. I tried with a bbox in WGS84, and it's still the whole world.

**Comment by matkoniecz** [comment]
> I looked at the rendering result from it, and it's the whole world. I tried with a bbox in WGS84, and it's still the whole world.

This is fixed by #151
